lib/addons/p5.dom.js

```diff
@@ -1745,6 +1745,7 @@
     this._cueIDCounter = 0;
     this._cues = [];
     this._pixelDensity = 1;
+    this.modified = false;
```

Comment:
Is `modified` a property that p5 users should be aware of/might use?

Consider `_modified` if not.